torchx/lib/torchx/backend.ex

```diff
@@ -143,6 +172,20 @@ defmodule Torchx.Backend do
     NIF.squeeze(to_ref(t)) |> from_ref(out)
   end
 
+  def expand(%T{} = t, shape) do
+    NIF.expand(to_ref(t), shape) |> from_bare_ref()
+  end
```

Comment:
Let's remove expand for now since we aren't going to use it and then we can merge it. :)